package.json

```diff
@@ -41,15 +41,15 @@
     "build:storybook": "cd docs && npm i && npm run build:storybook"
   },
   "dependencies": {
-    "@primer/primitives": "^9.0.3",
-    "@primer/view-components": "^0.34.0"
   },
   "devDependencies": {
     "@changesets/changelog-github": "^0.5.0",
     "@changesets/cli": "^2.27.1",
     "@csstools/postcss-sass": "^5.1.1",
     "@github/prettier-config": "^0.0.6",
+    "@primer/primitives": "^9.0.3",
```

Comment:
What do you think about moving this one to peerDependencies? does that loosen up the restrictions or does it not make a difference?

Reply:
Hmm... that would probably be fine until we release a major version of primitives. But hopefully that doesn't happen too often?